src/model.c

```diff
@@ -66,6 +64,8 @@ int ir_model_new(ir_model *model) {
 
 	ir_vector_init_lua(model->state);
 
+  ir_shader_init_lua(model->state);
```

Comment:
Not sure how this line changed. Seems like a minor thing but it's still worth mentioning for readability's sake.

Reply:
I moved out the binding of the `ir_shader_new_lua` function into a separate `ir_shader_init_lua` function together with the initialization of the shader metatable. That line wasn't there

Reply:
Ah. That'll do it. It should be fixed on the next `clang-format`.